api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -188,10 +187,12 @@ message ExtProcOverrides {
   bool async_mode = 2;
 
   // [#not-implemented-hide:]
-  // Set different optional properties than the default.
-  repeated string request_properties = 3;
+  // Set different optional attributes than the default setting of the
+  // *request_attributes* field.
```

Comment:
hi rather than `*` could you please use ` `` ` so that it appears as an `inline-literal`

same below

Reply:
just seeing that this is not-implemented, which wont be shown - but i guess the comment applies to the section above that is being removed from not-implemented

Reply:
Thanks -- I changed these as recommended. Yes, these two fields are still marked "not-implemented-hide." However, in anticipation of us un-hiding them in the future, I wanted to get the naming to be consistent.